src/services/reCaptcha.js

```diff
@@ -0,0 +1,24 @@
+const axios = require("axios");
+
+class ReCaptcha {
+  constructor(userToken) {
+    this.userToken = userToken;
+    this.url = "https://www.google.com/recaptcha/api/siteverify";
+    this.secret = process.env.RECAPTCHA_SECRET;
```

Comment:
Cool. See Kalvin's change in pull #293 for updating azure-pipeline.yaml to pass this through.